components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -150,18 +150,31 @@ __attribute__((objc_subclassing_restricted))
  then this color will be applied to the title.
  */
 @property(nonatomic, strong, nullable) UIColor *primaryTextColor;
```

Comment:
is `primaryTextColor` still required? It seems like it's better to have the new properties that assign values directly to the UI elements.

Reply:
Primary text color is required for the themer.